juniper_official/Interfaces/count-lo0-address-netconf.rule

```diff
@@ -0,0 +1,138 @@
+/*

```

Comment:
As we discussed move IPv4 and v6 loopback verifications into community

Reply:
moved file to community